lib/cdo/poste.rb

```diff
@@ -308,8 +308,9 @@ def inject_litmus_tracking(html, tracking_id, unique_id)
 
     def renderer
       @@renderer ||= begin
-        require 'cdo/pegasus/actionview_sinatra'
-        ActionView::Template.register_template_handler :md, ActionViewSinatra::MarkdownHandler
+        require 'action_view'
+        require 'cdo/markdown_handler'
+        MarkdownHandler.register unless MarkdownHandler.registered?
```

Comment:
I don't know if this "unless registered" logic is actually necessary, but figure it can't hurt